docs/TBG_macros.cfg

```diff
@@ -144,6 +144,8 @@ TBG_<species>_pngYX="--<species>_png.period 10 --<species>_png.axis yx --<specie
 # Enable macro particle merging
 TBG_<species>_merger="--<species>_merger.period 100 --<species>_merger.minParticlesToMerge 8 --<species>_merger.posSpreadThreshold 0.2 --<species>_merger.absMomSpreadThreshold 0.01"
 
+# Enable probalistic version of particle merging
+TBG_<species>_randomizedMerger="--<species>_randomizedMerger.period 100 --<species>_randomizedMerger.minParticlesToMerge 8 --<species>_randomizedMerger.ratioDeletedParticles 0.9"
```

Comment:
IMO it must be `minMacroParticlesToDivide` (same in line 145)

Reply:
I think in both plugins it should actually be `maxParticlesToMerge`, not `min`.

Reply:
> maxParticlesToMerge

in this case, it should be maxMacroParticlesToMerge, because we merge only macroparticles

Reply:
Perhaps. But other parameters use particles, not macroparticles.

Reply:
Yes, but this parameter in macroparticles

Reply:
Yes, like every other parameter in this plugin.